core/domain/email_manager.py

```diff
@@ -329,7 +329,7 @@
         'There are suggestions on the <a href="%s%s">Contributor Dashboard</a> '
         'that have been waiting for more than %s days for review. Please take '
         'a look at the suggestions mentioned below and help them get reviewed '
-        'by going to the <a href="%s%s#/roles">admin roles page</a> and either:'
+        'by going to the <a href="http://www.oppia.org/contributor-dashboard-admin%s">admin roles page</a> and either:'
```

Comment:
This link should be https.

Also I am concerned that you've replaced a templated string with a hardcoded string. What gets substituted in the two %s's in the original message and how have you verified/tested that what you're doing here leads to the correct link? (This is a backend change so the frontend tests are not relevant here.)